### PR TITLE
CUDOS-887: Filling in the delegators table with data fetched from the server instead of dummy data

### PR DESCRIPTION
**Pull request by stanfilip0v**
CUDOS-887: Filling in the delegators table with data fetched from the server instead of dummy data

### CODE REVIEW

**Review by stefanpetkov90 — APPROVED** [review]

**Review by SpaghettiOverload — APPROVED** [review]

File: src/containers/ValidatorDetails/components/Delegators/components/Delegations/index.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+  let component = null
+
+  if (delegations.loading) {
+    component = (
+      <Box display="flex" justifyContent="center">
+        <CircularProgress color="primary" variant="indeterminate" />
+      </Box>
+    )
+  } else if (!items.length) {
```
Comment (maptuhec) [inline src/containers/ValidatorDetails/components/Delegators/components/Delegations/index.tsx]: is it possible to make this structure more readable?

File: src/containers/ValidatorDetails/components/Delegators/components/Redelegations/index.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+  let component = null
+
+  if (redelegations.loading) {
+    component = (
+      <Box display="flex" justifyContent="center">
+        <CircularProgress color="primary" variant="indeterminate" />
+      </Box>
+    )
+  } else if (!items.length) {
```
Comment (maptuhec) [inline src/containers/ValidatorDetails/components/Delegators/components/Redelegations/index.tsx]: same here

**Review by maptuhec — APPROVED** [review]

File: src/containers/ValidatorDetails/components/Delegators/components/Delegations/index.tsx (hunk 1 above)
Comment (stanfilip0v) [reply src/containers/ValidatorDetails/components/Delegators/components/Delegations/index.tsx]: removed the "if else" block

File: src/containers/ValidatorDetails/components/Delegators/components/Redelegations/index.tsx (hunk 2 above)
Comment (stanfilip0v) [reply src/containers/ValidatorDetails/components/Delegators/components/Redelegations/index.tsx]: removed the "if else" block

**Pull request merged**